src/clients/schemas.ts

```diff
@@ -111,6 +111,31 @@ const ConfigValidateSchema = z.object({
   'source-yaml': z.string(),
 });
 
+const RunPipelineResponseSchema = z.object({
+  number: z.number(),
+});
+
+const ProjectSchema = z.object({
+  id: z.string(),
+});
+
+const PipelineDefinitionSchema = z.object({
+  id: z.string(),
+  name: z.string(),
```

Comment:
example response can be found in our docs here: https://circleci.com/docs/api/v2/index.html#tag/Pipeline/operation/listPipelineDefinitions

we may want more of those fields in the future, but right now we're only using id and name so we don't need to validate the others at the moment